code/modules/mob/living/carbon/xenomorph/castes/drone/castedatum_drone.dm

```diff
@@ -60,7 +60,6 @@
 		/datum/action/xeno_action/activable/transfer_plasma/drone,
 		/datum/action/xeno_action/activable/salvage_plasma,
 		/datum/action/xeno_action/activable/corrosive_acid/drone,
-		/datum/action/xeno_action/activable/larval_growth_sting,
 		/datum/action/xeno_action/toggle_pheromones,
```

Comment:
Why are you removing larval from abilities list
Leave it in as a infection booster chem